### DISCUSSION TITLE
We should have a code of conduct for runbook

### DISCUSSION THREAD

**Discussion by LeoQuote** [prometheus-operator/kube-prometheus#1098] [Ideas]
I can always see this label for my alert, when I click it, it shows a github page waiting to be created and drafted.
runbook_url: https://github.com/prometheus-operator/kube-prometheus/wiki/cputhrottlinghigh

I am really not sure whether should I draft a runbook on my own, is it good or bad, it will be better if there's a code of conduct in wiki home https://github.com/prometheus-operator/kube-prometheus/wiki , so people will know the basic principle of writing a runbook.

**Comment by paulfantom** [comment]
Great idea! Do you mind drafting such CoC?

---
> I can always see this label for my alert, when I click it, it shows a github page waiting to be created and drafted.

Yes, our idea was to point to a blank page to invite users create runbooks for alerts when they fire. We do have a template for a new runbook in https://github.com/prometheus-operator/kube-prometheus/wiki/Runbooks, but sadly I don't see a way to allow this to be a template for new page :(

**Reply by LeoQuote** [reply]
I cannot see it either, but maybe we can create all web page as the template, people who click the page can see the CoC and guide right away, so people wont get puzzled

**Reply by LeoQuote** [reply]
Or a very simple notice to guide people to edit this page or see the CoC for some guideline

**Reply by paulfantom** [reply]
The wiki is meant only for those runbooks, so if there is a possibility to have templates for all pages, I am fine with it.

**Reply by LeoQuote** [reply]
I mean... creating all pages that not exist for now to be identical to the "template", but it's a bit hard...

**Reply by paulfantom** [reply]
This is impossible as we would need somehow to keep track of all alerts we have and of every update to those alert names. And even then there is a possibility someone will get to an empty page.

**Reply by paulfantom** [reply]
Right now I added a footer page, which should help a bit. However, it doesn't solve the problem of what to do with a blank new page.
